crates/chronicle/src/bootstrap/mod.rs

```diff
@@ -313,13 +313,10 @@ where
             None
         };
 
-        let id_pointer = if matches.is_present("anonymous-api") {
-            None
-        } else {
-            matches
-                .value_of("id-pointer")
-                .map(|id_pointer| id_pointer.to_string())
-        };
+        // Defaults to look for "/sub"
```

Comment:
```suggestion
        // CliModel defaults to look for "/sub"
```
Something like that might help piece things together if the code in `cli.md` is changed by somebody unaware of this comment.